addons/l10n_it_edi/tests/test_edi_import.py

```diff
@@ -71,6 +72,28 @@ def test_receive_signed_vendor_bill(self):
             }],
         }])
 
+    def test_receive_bill_sequence(self):
+        """ Ensure that the received bill gets assigned the right sequence. """
+        def mock_commit(self):
+            pass
+
+        filename = 'IT01234567890_FPR02.xml'
+        with (patch.object(self.proxy_user.__class__, '_decrypt_data', return_value=self.fake_test_content),
+              patch.object(sql_db.Cursor, "commit", mock_commit),
+              tools.mute_logger("odoo.addons.l10n_it_edi.models.account_move"),
+              freeze_time('2019-01-01')):
+            self.env['account.move']._l10n_it_edi_process_downloads({
```

Comment:
```suggestion
            self.env['account.move'].with_company(self.company)._l10n_it_edi_process_downloads({
```

Reply:
Please use the accounting test company, it's set in the superclass, don't rely on the environment company.